docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# will be deprecated when https://github.com/zotero/translation-server-v2/issues/1 will be released
+echo "Updating zotero translators"
+cd /app/modules/translators/
+git pull origin master
+
+cd /app/
+exec npm start
```

Comment:
Does that work? After the first `cd` we are not in the main directory anymore and therefore the second `cd` should IMO fail. The name of this script `docker-entrypoint.sh` does not look suitable for me.

Reply:
No, the `cd` is fine — it's an absolute path (though it could just be `cd /app`).

Reply:
Ah my mistake on that, sorry. But the name of this script is still debatable.

Reply:
I think "docker-entrypoint.sh" naming is often used when dockerizing an application.
It could of course be changed to anything else.

Reply:
I would expect that this script can even be called outside of docker. No?

@dstillman Should decide about the name here.

Reply:
The name is fine.